src/Roslyn.Suppressions/Roslyn.Suppressions/SettingsFile/RoslynSettingsFileInfo.cs

```diff
@@ -32,9 +32,9 @@ internal static class RoslynSettingsFileInfo
         /// </summary>
         public static string GetSettingsFilePath(string sonarProjectKey)
         {
-            var escapedName = PathHelper.EscapeFileName(sonarProjectKey);
+            var escapedName = PathHelper.EscapeFileName(NormalizeKey(sonarProjectKey));
```

Comment:
I'm not sure this is actually required currently if the dictionary is configured with the correct string comparer on constructor.

It won't hurt to force to lower case, but the `NormalizeKey` method doesn't need to be public.

Reply:
I think it's better to have it lowered and be consistent and use the same key everywhere.